### PR TITLE
images: fix templates/ISOs all filter

### PR DESCRIPTION
**Pull request by shwstppr**
Screenshot from 2020-08-31 18-35-37

Show **All** filter menu option for admins

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan package

@rhtyd @davidjumani please review

File: src/views/AutogenView.vue [hunk 1]
```diff
@@ -45,11 +45,11 @@
                 <a-select
                   v-if="!dataView && $route.meta.filters && $route.meta.filters.length > 0"
                   :placeholder="$t('label.filterby')"
-                  :value="$route.query.filter || (['Admin', 'DomainAdmin'].includes($store.getters.userInfo.roletype) && $route.name === 'vm' ? 'all' : 'self')"
+                  :value="$route.query.filter || (['Admin', 'DomainAdmin'].includes($store.getters.userInfo.roletype) && ['vm', 'iso', 'template'].includes($route.name) ? 'all' : 'self')"
```
Comment (davidjumani) [inline src/views/AutogenView.vue:48]: @shwstppr It shows all but the api calls `templatefilter=self`. Could you fix that ?

**Comment by rhtyd** [comment]
LGTM, I'll wait for @davidjumani 's testing and merge this tomorrow

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by davidjumani — APPROVED** [review]
LGTM Thanks @shwstppr 
@rhtyd Good to go!

**Comment by shwstppr** [comment]
@rhtyd @davidjumani just checked there was a minor User role. Do we need to revert this or can I raise a new PR?

**Pull request merged**